todo/src/components/TodoList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Todo from './Todo';
+
+const TodoList = (props) => {
+
+  return(
+    <ul>
+      {props.todos.map((task, index) => { return <Todo key={index} todo={task.value} completed={task.completed} id={task.id} />})}
```

Comment:
No need for a `return` in a one-liner.
```suggestion
      {props.todos.map((task, index) => <Todo key={index} todo={task.value} completed={task.completed} id={task.id} />)}
```